code/datums/mutations/actions.dm

```diff
@@ -190,3 +190,38 @@
 /obj/effect/proc_holder/spell/self/void/cast(mob/user = usr)
 	. = ..()
 	new /obj/effect/immortality_talisman/void(get_turf(user), user)
+
+/datum/mutation/human/self_amputation
+	name = "Autotomy"
+	desc = "Allows a creature to voluntary discard a random appendage."
+	quality = MINOR_NEGATIVE //upsides and downsides
+	text_gain_indication = "<span class='notice'>Your joints feel loose.</span>"
+	instability = 30
+	power = /obj/effect/proc_holder/spell/self/self_amputation
+
+/obj/effect/proc_holder/spell/self/self_amputation
+	name = "Drop a limb"
+	desc = "Concentrate to make a random limb pop right off your body."
+	clothes_req = FALSE
+	human_req = FALSE
+	charge_max = 100
```

Comment:
Should have a really really small recharge time so you can just shoot limbs off in every direction at will